extensions/wikia/AdEngine/ResourceLoaders/ResourceLoaderAdEngineBase.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+abstract class ResourceLoaderAdEngineBase extends ResourceLoaderModule {
+	const TTL_SCRIPTS = 1800; // half an hour -- cache scripts from ad.71i.de for this time
```

Comment:
I did the same mistake ;D
https://github.com/Wikia/app/pull/7977#discussion-diff-36091167